cou/utils/juju_utils.py

```diff
@@ -36,11 +38,11 @@
 )
 
 JUJU_MAX_FRAME_SIZE: int = 2**30
-DEFAULT_TIMEOUT: int = 60
+DEFAULT_TIMEOUT: int = int(os.environ.get("COU_TIMEOUT", 10))
 DEFAULT_MAX_WAIT: int = 5
 DEFAULT_WAIT: float = 1.1
-DEFAULT_MODEL_RETRIES: int = 30
-DEFAULT_MODEL_RETRY_BACKOFF: int = 2
+DEFAULT_MODEL_RETRIES: int = int(os.environ.get("COU_MODEL_RETRIES", 5))
+DEFAULT_MODEL_RETRY_BACKOFF: int = int(os.environ.get("COU_MODEL_RETRY_BACKOFF", 2))
```

Comment:
Can we open an issue to document those env vars just to not lose track?

Reply:
I working on documentation so I will mention it there. Not sure if these two are needed, but I can add them.

Reply:
As discussed yesterday, you said that you are in charge for documentation and you won't miss this, so resolve the conversation